src/main/resources/org/jenkins/plugins/lockableresources/actions/LockableResourcesRootAction/index_sk.properties

```diff
@@ -22,7 +22,10 @@
 
 #headers
 header.resources=Uzamykate\u013en\u00e9 zdroje
-header.labels=Popisky
+header.labels=\u0160t\u00edtky
+#warnig resources not configured
+resources.not_configured=There are no resources configured at the moment.
```

Comment:
Ping for my, need to be translated

Reply:
Translated , I will wait till crowdin update this branch

Reply:
I'm not so sure Crowdin has been configured for any other branch than master... 🤔

Reply:
What you means? Last changes LGTM. It is translated as well

Reply:
I was wrong then. 🤷

Reply:
In that case it can be merged in master. Thx